src/Settings.cpp

```diff
@@ -489,6 +489,14 @@ QColor Settings::getDefaultColorValue(const std::string& group, const std::strin
     return QColor();
 }
 
+void Settings::clearRecentFiles ()
```

Comment:
Why not a general `void clear(const std::string& group, const std::string& name);`?

Reply:
Do you mean to take a parameter as a parameter instead of using a literal string in a function like the code below?
```
void Settings::clearRecentFiles(const std::string& group, const std::string& name) {  ...
  // call beginGroup() and remove method with parametrs(group, name)
}
```

Reply:
Yes, but the function name should be general, like clearValue().

Reply:
I think resolve this conversation by ec887dfbbd92226c008cf9696a5457ea804d41de.